### DISCUSSION TITLE
[code action][edit this page] add online editors as an option

### DISCUSSION THREAD

**Discussion by copdips** [squidfunk/mkdocs-material#6748] [Ideas]
Hello,

Current `edit this page` button in code action redirects to the orignal edit url.
As we also have some online editors like vscode.dev, etc., is it possible to add them as an option ?

For example, the edit button on [this page](https://code.visualstudio.com/docs/setup/setup-overview) redirects to:
https://vscode.dev/github/microsoft/vscode-docs/blob/main/docs/setup/setup-overview.md

**Comment by markallasread** [comment]
Seems like an interesting possibility, this discussion offers some insight #3242  — the 'edit_uri' is adjustable.
You might want to bring this up on the mkdocs respository itself, someone might be happy to flesh out a plugin that allows a multitude of editors to be listed.

**Comment by squidfunk** [comment]
Thanks for suggesting. This is more or less beyond the scope of this project, since those editors will be added through JavaScript anyway ☺️ You could just add them with some [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript) to an existing site without any need for us to change the project.
